minecraft_servers/seaside/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "seaside",
+  "nice_name": "SeaSide",
+  "direct_ip": "mc-seaside.eu",
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "server_wildcards": [
+    "%.mc-seaside.eu",
+    "37.114.56.122"
+  ],
+  "social": {
+    "web_shop": "https://seaside.craftingstore.net",
+    "twitter": "SeaSide33692248",
+    "youtube": "https://www.youtube.com/channel/UCSamt0KtmADC77AIC4_-0XQ",
+    "discord": "https://discord.gg/PranAzf",
+    "instagram": "mc.seaside"
+  },
+  "discord": {
+    "server_id": 704281126855901215,
+    "rename_to_minecraft_name": true
```

Comment:
Only allowed for partners, please use `false`, as this is the default value.

Reply:
should be fixed